buildpipeline/pipeline.json

```diff
@@ -45,6 +45,23 @@
             "Type": "build/product/",
             "ConfigurationGroup": "Release"
           }
+        },
+        {
+          "Name": "DotNet-CoreFx-Trusted-Linux-RedHat6",
```

Comment:
Thanks @weshaggard . Got your point, same BD name, but different docker and queue name for RHEL6. Changed as you suggested.